scripts/update_images.py

```diff
@@ -4,21 +4,87 @@
 MANAGER_PATH = "config/manager/manager.yaml"
 REPLACE_COMPONENTS_IMAGES_PATH = "tests/replace_components_images.sh"
+HELM_INSTALL_SCRIPT_PATH = "tests/helm_install.sh"
 BASH_HEADER = "#!/usr/bin/env bash\n\n"
 
+HELM_INSTALL_COMMAND_HEADER = """readonly ROOT_DIR="$(dirname "$(dirname "${0}")")"
+
+helm install test-openshift sumologic/sumologic \\
+  --set sumologic.accessId="dummy" \\
+  --set sumologic.accessKey="dummy" \\
+  --set sumologic.endpoint="http://receiver-mock.receiver-mock:3000/terraform/api/" \\
+  --set sumologic.scc.create=true \\
+  --set fluent-bit.securityContext.privileged=true \\
+  --set kube-prometheus-stack.prometheus-node-exporter.service.port=9200 \\
+  --set kube-prometheus-stack.prometheus-node-exporter.service.targetPort=9200 \\
+  --set fluentd.logs.containers.multiline.enabled=false \\
+  --set metrics-server.enabled=true \\
+  --set metrics-server.apiService.create=false \\
+  --set otelagent.enabled=true \\
+  --set telegraf-operator.enabled=true \\
+  --set falco.enabled=true \\
+  --set tailing-sidecar-operator.enabled=true \\
+  --set opentelemetry-operator.enabled=true \\
+  --version 2.19.1 \\
+  -n sumologic-system \\
+  --create-namespace -f "${ROOT_DIR}/tests/values.yaml" \\\n"""
+
+# COMPONENTS_CONFIG_MAP maps helm chart configuration keys into components names
+COMPONENTS_CONFIG_MAP = {
+    "instrumentation.instrumentationJobImage.image": "kubernetes-tools-kubectl",
+    "kube-prometheus-stack.kube-state-metrics.image": "kube-state-metrics",
+    "kube-prometheus-stack.prometheus-node-exporter.image": "node-exporter",
+    "kube-prometheus-stack.prometheus.prometheusSpec.image": "prometheus",
+    "kube-prometheus-stack.prometheusOperator.image": "prometheus-operator",
+    "kube-prometheus-stack.prometheusOperator.prometheusConfigReloader.image": "prometheus-config-reloader",
+    "kube-prometheus-stack.prometheusOperator.prometheusConfigReloaderImage": "prometheus-config-reloader",
+    "kube-prometheus-stack.prometheusOperator.thanosImage": "thanos",
+    "metadata.image": "sumologic-otel-collector",
+    "metrics-server.image": "metrics-server",
+    "opentelemetry-operator.kubeRBACProxy.image": "kube-rbac-proxy",
+    "opentelemetry-operator.kubeRBACProxy.image.repository": "kube-rbac-proxy",
+    "opentelemetry-operator.manager.autoInstrumentationImage.dotnet": "autoinstrumentation-dotnet",
+    "opentelemetry-operator.manager.autoInstrumentationImage.java": "autoinstrumentation-java",
+    "opentelemetry-operator.manager.autoInstrumentationImage.nodejs": "autoinstrumentation-nodejs",
+    "opentelemetry-operator.manager.autoInstrumentationImage.python": "autoinstrumentation-nodejs",
+    "opentelemetry-operator.manager.collectorImage": "sumologic-otel-collector",
+    "opentelemetry-operator.manager.image": "opentelemetry-operator",
+    "opentelemetry-operator.manager.image.repository": "opentelemetry-operator",
```

Comment:
Isn't a conflict here?

Reply:
fixed in https://github.com/SumoLogic/sumologic-kubernetes-collection-helm-operator/pull/121/commits/12be38b012bc0f4f270cdc52c596ffecadb82b91 
list of image keys which aren't in values.yaml was incorrect